docs/DeveloperGuide.adoc

```diff
@@ -75,21 +83,27 @@ The sections below give more details of each component.
 === UI component
 
 .Structure of the UI Component
+
+[NOTE]
+UI image and description to be updated with the later milestones.
+
 image::UiClassDiagram.png[]
```

Comment:
![Screenshot 2019-10-25 at 1 26 12 PM](https://user-images.githubusercontent.com/42567064/67545265-1afdb600-f72b-11e9-8947-4ee980194a04.png)
Lacks Figure Label. Format can be made consistent as with the previous diagram. 

Should TeamListPanel, MentorListPanel etc be under the UI box?